devices.js

```diff
@@ -1443,6 +1443,13 @@ const devices = [
         description: 'Hue Bloom',
         extend: hue.light_onoff_brightness_colorxy,
     },
+    {
+        zigbeeModel: ['LCP001', 'LCP002'],
+        model: '4090331P9',
+        vendor: 'Philips',
+        description: 'Hue Ensis',
+        extend: hue.light_onoff_brightness_colorxy,
```

Comment:
Doesn't it also support color temperature? (try with `hue.light_onoff_brightness_colortemp_colorxy`)

Reply:
Yes, you are right. I've used `hue.light_onoff_brightness_colortemp_colorxy` locally for some time, but made a cut and paste error when making the same change in the git repository that I pushed.